src/Stratis.Bitcoin.Features.PoA/PoABlockDefinition.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Microsoft.Extensions.Logging;
+using NBitcoin;
+using Stratis.Bitcoin.Consensus;
+using Stratis.Bitcoin.Features.MemoryPool;
+using Stratis.Bitcoin.Features.MemoryPool.Interfaces;
+using Stratis.Bitcoin.Features.Miner;
+using Stratis.Bitcoin.Mining;
+using Stratis.Bitcoin.Utilities;
+
+namespace Stratis.Bitcoin.Features.PoA
+{
+    public class PoABlockDefinition : BlockDefinition
+    {
+        /// <summary>Instance logger.</summary>
+        private readonly ILogger logger;
+
+        public PoABlockDefinition(
+            IConsensusManager consensusManager,
+            IDateTimeProvider dateTimeProvider,
+            ILoggerFactory loggerFactory,
+            ITxMempool mempool,
+            MempoolSchedulerLock mempoolLock,
+            Network network)
+            : base(consensusManager, dateTimeProvider, loggerFactory, mempool, mempoolLock, new MinerSettings(), network)
+        {
+            this.logger = loggerFactory.CreateLogger(this.GetType().FullName);
+        }
+
+        /// <inheritdoc/>
+        public override void AddToBlock(TxMempoolEntry mempoolEntry)
+        {
+            this.AddTransactionToBlock(mempoolEntry.Transaction);
+            this.UpdateBlockStatistics(mempoolEntry);
+            this.UpdateTotalFees(mempoolEntry.Fee);
+        }
+
+        /// <inheritdoc/>
+        public override BlockTemplate Build(ChainedHeader chainTip, Script scriptPubKey = null)
+        {
+            this.OnBuild(chainTip, new Script());
+
+            this.coinbase.Outputs[0].ScriptPubKey = new Script();
+            this.coinbase.Outputs[0].Value = Money.Zero;
+
+            return this.BlockTemplate;
+        }
+
+        /// <inheritdoc/>
+        public override void UpdateHeaders()
+        {
+            base.UpdateBaseHeaders();
+
+            this.block.Header.Bits = Target.Difficulty1; // TODO POA use some constant difficulty all the time so chainwork rises
+            // TODO maybe make it a consensus rule to ensure that bits are always constant (!!!!!!)
+            // If we do that we can reuse selecting chain using best chainwork
```

Comment:
Maybe put the constant difficulty in the Consensus Options (yeah to make a rule sounds definitely correct.

Reply:
In the next PR adding it to rule